packages/prisma/migrations/20240522085944_add_template_direct_links/migration.sql

```diff
@@ -0,0 +1,41 @@
+/*
+  Warnings:
+
+  - Added the required column `source` to the `Document` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- CreateEnum
+CREATE TYPE "DocumentSource" AS ENUM ('DOCUMENT', 'TEMPLATE', 'TEMPLATE_DIRECT_LINK');
+
+-- AlterTable
+ALTER TABLE "Document" ADD COLUMN     "source" "DocumentSource";
+
+-- Custom: UpdateTable
+UPDATE "Document" SET "source" = 'DOCUMENT' WHERE "source" IS NULL;
+
+-- Custom: AlterColumn
+ALTER TABLE "Document" ALTER COLUMN     "source" SET NOT NULL;
```

Comment:
This was migration was altered to make `document.source` not null, but at the same time set all current sources to `document`